webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -1192,6 +1192,15 @@ def mocked_get(url, params, **options):
         response = self.client.get(url, {'page': 'meow'})
         eq_(response.status_code, 200)
 
+        # If you cease to be active, access should be revoked automatically
+        user = User.objects.get(id=user.id)
+        user.is_active = False
+        user.save()
+        assert not user.has_perm('crashstats.view_exploitability')
+        response = self.client.get(url)
+        ok_(settings.LOGIN_URL in response['Location'] + '?next=%s' % url)
```

Comment:
So that means, if a user is made inactive, they will be forever redirected to the login page? What happens when an inactive user tries to log in?